oioioi/participants/controllers.py

```diff
@@ -209,6 +209,16 @@ def is_registration_open(self, request):
             auditLogger.warning("RegistrationAvailabilityConfig does not exist for contest %s", request.contest)
             return True
 
+    def registration_status(self, request):
```

Comment:
create an issue about removing function can_register and replacing it with registration_status

Reply:
issue: #463